va/va_vpp.h

```diff
@@ -1447,6 +1449,102 @@ typedef struct _VAProcFilterCap3DLUT {
     uint32_t            va_reserved[VA_PADDING_HIGH];
 } VAProcFilterCap3DLUT;
 
+/**
+ *  \brief Video Processing Super Resolution Predefined Mode.
+ *
+ *  Super Resolution is the process of recovering a High Resolution(HR) image/video
+ *  from a given Low Resolution(LR) image/video. A few modes are predefined for
+ *  different scenarios.
+ */
+typedef enum _VAProcSuperResolutionMode {
+    /**
+     * \brief Super Resolution Default Mode.
+     * This mode applies to video playback (especially natural video).
```

Comment:
1,maybe changes to "this mode is tuned for video playback"? or "recommended to be used" as  next two mode.  and app always could use this mode for different usage scenarios , right? 
2, the default mode is not one of the other modes?

Reply:
> 1,maybe changes to "this mode is tuned for video playback"? or "recommended to be used" as next two mode. and app always could use this mode for different usage scenarios , right? 2, the default mode is not one of the other modes?

#1 Modified.
#2 Yes, the default is not one of the other modes.

Reply:
for #2, it sounds this behavior doesn't keep consistency with other flags in VAAPI. The default is one of available modes for other flags.